Makefile

```diff
@@ -80,7 +128,12 @@ build: build-yarn
 
 .PHONY: build-solidity-types
 build-solidity-types: ## generate Solidity typings
-	yarn workspace @hoprnet/hopr-ethereum run build:sol:types
+	echo "Foundry create binding"
+	$(MAKE) -C packages/ethereum/contracts/ overwrite-sc-bindings
+# Change git = "http://..." into version = "1.0.2"
+	sed -i -e 's/https:\/\/github.com\/gakonst\/ethers-rs/1.0.2/g' ${CURDIR}/packages/ethereum/crates/bindings/Cargo.toml
+	sed -i -e 's/git/version/g' ${CURDIR}/packages/ethereum/crates/bindings/Cargo.toml
+	echo -e "\n[lib] \ncrate-type = [\"cdylib\", \"rlib\"] # rlib is necessary to run integration tests" >> ${CURDIR}/packages/ethereum/crates/bindings/Cargo.toml
```

Comment:
Looks like the comment breaks the echo appending to the file here?

Reply:
The line with code (like below)
```
[lib] 
crate-type = ["cdylib", "rlib"] # rlib is necessary to run integration tests
```
is supposed to added into the Cargo.toml file.
But I can just remove the `#` and comment the action

Reply:
oh I get it, then leave it as is